Kuzzle/API/Controllers/AuthController.cs

```diff
@@ -144,6 +150,9 @@ JObject credentials
 
       api.AuthenticationToken = (string)response.Result["jwt"];
 
+      if (credentials["username"] != null)
```

Comment:
you need to decode the jwt and get the kuid instead, username is only used by the local auth plugin

Reply:
This now use the KUID from the auth:login response instead of the username